apps/frontend/src/modules/contributors/slices/contributor.slice.ts

```diff
@@ -4,16 +4,18 @@ import { DataStatus } from "~/libs/enums/enums.js";
 import { type ValueOf } from "~/libs/types/types.js";
 
 import { type ContributorGetAllItemResponseDto } from "../libs/types/types.js";
-import { loadAll } from "./actions.js";
+import { loadAll, loadAllByProjectId } from "./actions.js";
 
 type State = {
 	contributors: ContributorGetAllItemResponseDto[];
 	dataStatus: ValueOf<typeof DataStatus>;
+	projectContributors: ContributorGetAllItemResponseDto[];
 };
 
 const initialState: State = {
 	contributors: [],
 	dataStatus: DataStatus.IDLE,
+	projectContributors: [],
```

Comment:
This should be inside project slice and related actions too

Reply:
Good point